README.md

```diff
@@ -110,6 +110,7 @@ Configuration is achieved through environment variables.
 | `FTL_SERVICE_CONNECTION` | `DUMMY`: (default) Dummy service connection <br />`GLIMESH`: Glimesh service connection <br />`REST`: REST service connection ([docs](docs/REST_SERVICE.md)) | This configuration value determines which service FTL should plug into for operations such as stream key retrieval. |
 | `FTL_SERVICE_METADATAREPORTINTERVALMS` | Time in milliseconds | Defaults to `4000`, controls how often FTL stream metadata will be reported to the service. |
 | `FTL_MAX_ALLOWED_BITS_PER_SECOND` | Integer bits per second | Defaults to `0` (disabled), FTL connections that exceed the bandwidth specified here will be stopped.<br />**Note that this is a strictly enforced maximum** based on a rolling average; consider providing some buffer size for encoder spikes above the configured average. |
+| `FTL_SERVICE_NACK_LOST_PACKETS` | `0`: Ignore missing packets <br />`1`: Send NACKs | Determines whether to send NACKs for missing packet sequences in the stream. |
```

Comment:
`FTL_SERVICE*` is normally for ServiceConnections - perhaps just `FTL_NACK_LOST_PACKETS` ?